backend/tests/unit/book.test.ts

```diff
@@ -39,6 +43,16 @@ describe('Fetch multiple books', () => {
   });
 });
 
+describe('Search through all books', () => {
+  test('Make a search', async () => {
+    const books = await searchBooks('hitchhiker', 0, 5);
+
+    expect(books).not.toBe(null);
+    expect(books![0].isbn).toBe('0517226952');
+    expect(books?.length).toBe(5);
```

Comment:
i feel like this is a wierd test since our data could change which could cause these to be different. Maybe use arraycontaining instead of array at index.tobe(value). and also might be better to check if thier is at least one result instead of a static always 5 results.

Reply:
Resolved in commit 925e7f8